libraries/heft-config-file/src/ConfigurationFile.ts

```diff
@@ -87,7 +96,7 @@ interface IConfigurationFileFieldAnnotation<TField> {
  *
  * @beta
  */
-export interface IJsonPathMetadata {
+export interface ICustomJsonPathMetadata {
   /**
    * If `IJsonPathMetadata.pathResolutionMethod` is set to `PathResolutionMethod.custom`,
```

Comment:
```suggestion
   * If `ICustomJsonPathMetadata.pathResolutionMethod` is set to `PathResolutionMethod.custom`,
```

Reply:
Ehhh, maybe not given the type is still exported below. Just a bit funky for the comment to be referencing a different interface.